### ISSUE TITLE
Mark best / outstanding papers

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-anthology#240]
It would be nice to mark best and outstanding papers and other awards, both in the list view and on the paper page. Maybe with a yellow star: ⭐.

**Comment by evanmiltenburg** [comment]
COLING 2018: http://coling2018.org/coling-2018-best-papers/

**Comment by mbollmann** [comment]
Do we mark best papers in the XML, or add another YAML source? I think I'd lean towards the former.

**Comment by mjpost** [comment]
Yes, in the XML. And we need to be able to handle multiple kinds of awards (best student paper, outstanding paper, test of time, etc). What do you think about this? Maybe

    <award>Outstanding paper</award>

**Comment by pkolachi** [comment]
Hi, I can take this up in the rest of this week -- if there is a consensus on how to address this.

**Comment by mjpost** [comment]
That would be awesome! I suggest adding an “award” tag as mentioned above. The hard part is making the annotations. Once they have been made, we could change the format easily with a script.

**Comment by mjpost** [comment]
Another list: https://naacl2018.wordpress.com/2018/03/22/test-of-time-award-papers/

**Comment by pkolachi** [comment]
Thanks. I'm making progress with this by extending the information in the ACL wiki with more information from archive conference pages. That followed by verifying against the front-matter  corresponding to a conference. There are some discrepancies (conflicts) and I don't know what's the best say to resolve these. ACL14 may be an example, where there wasn't a best short paper awarded according to the program, but the ACL wiki says otherwise. 
Personally, I am all for crowd-fixing any conflicts like these and simply putting together all the information available, but are there better recommendations here?

**Comment by mjpost** [comment]
I don’t know the right answer to ACL 2014 but I would prioritize the most local information, so in this case, the program. You could note discrepancies in a commit message. They can always be fixed later.

**Comment by akoehn** [comment]
It's several years, but ACL 2014 was actually the first big conference I went to and I only remember two awards.  [The website says so as well](http://acl2014.org/BestPaper.htm)
The "ACL 14 best short paper" in the wiki is only listed as an honorable mention on the official website.
[The official handbook](http://acl2014.org/handbook-acl2014.pdf) also only lists two: best long paper and best student paper.

**Comment by mjpost** [comment]
"Outstanding paper", "honorable mention", and others should also be annotated. This is why we have an arbitrary text string for the award.

**Comment by pkolachi** [comment]
Yup, making a tsv table with paperid, title, and the award string that's mentioned anywhere. This is incredibly fun :)

**Comment by mjpost** [comment]
Hi @pkolachi, are you still interested in doing this or did you lose interest? Even just having a CSV / TSV file of the format:

    P19-1426	Best Paper

would do the trick.

**Comment by mjpost** [comment]
Many of these were added in #954. [This spreadsheet](https://docs.google.com/spreadsheets/d/174uNP-wpyjU0Hx4CsT_tQPiDmU5b-5jHzCaEJMYfXZA/edit?usp=sharing) tracks progress.

**Comment by mbollmann** [comment]
> Many of these were added in #954. [This spreadsheet](https://docs.google.com/spreadsheets/d/174uNP-wpyjU0Hx4CsT_tQPiDmU5b-5jHzCaEJMYfXZA/edit?usp=sharing) tracks progress.

Is the spreadsheet you linked here still up-to-date, @mjpost?

**Comment by mjpost** [comment]
I think so, but it is a first-pass effort, and I think everything needs to be validated. I've shared it with you and added a "verified" column in case your interest is piqued.

**Comment by nschneid** [comment]
#6025

**Comment by nschneid** [comment]
Does https://aclanthology.org/info/contrib/ need to be updated with current processes for submitting paper awards? I take it these are not generally available at ingestion time.
